app/terminal/terminal.py

```diff
@@ -20,6 +20,7 @@ def do_help(self):
         self._sout.write('\t'+'qu: queue an operation to be started later (required) operation name, adversary ID and group ID (optional) jitter fraction' + '\n')
         self._sout.write('\t'+'re: view a specific result (required) link ID - from the decision chain.' + '\n')
         self._sout.write('\t'+'st: start an operation. (required) a queued operation ID.' + '\n')
+		self._sout.write('\t'+'bl: Import blacklist of facts via a csv file with columns [fact, value, score]. Command usage: bl [csv file path] [op_id]' + '\n')
```

Comment:
remove this line - this is not a runnable function

Reply:
Done